src/test/java/net/sf/jabref/logic/exporter/MetaDataSerializerTest.java

```diff
@@ -48,7 +49,18 @@ public void serializeSingleSaveAction() {
     }
 
     @Test
-    public void emptyGroupsIfNotSet() {
-        assertEquals(Optional.empty(), metaData.getGroups());
+    public void serializeSingleContentSelectors() {
+        List<String> values = new ArrayList(4);
+        values.add("approved");
+        values.add("captured");
+        values.add("received");
+        values.add("status");
+
+        metaData.addContentSelector(new ContentSelector("status", values));
+
+        Map<String, String> expectedSerialization = new TreeMap<>();
+        expectedSerialization.put(MetaData.SELECTOR_META_PREFIX + "status",
```

Comment:
Inline `MetaData.SELECTOR_META_PREFIX` so that it is clear if e.g. a underscore is added or not.